frontend/packages/shipwright-plugin/integration-tests/features/shipwright-build-in-topology.feature

```diff
@@ -7,65 +7,80 @@ Feature: Shipwright build in topolgy
               And user is at developer perspective
               And user has created or selected namespace "aut-shipwright-build-details"
               And user has installed Shipwright Operator
+              And user has installed OpenShift Serverless Operator
               And user is at Add page
-              And user has created shipwright builds
+              And user has created shipwright build-strategies
 
 
-        @smoke @to-do
-        Scenario Outline: Sidebar for workload with shipwright build: SWB-02-TC01
+        @smoke
+        Scenario Outline: Topology page in dev perspective: SWB-02-TC01
             Given user has created workload using yaml "<workload_yaml>"
-              And user is at the Topology page
-             When user clicks on workload "<workload_name>"
-              And user clicks on Resource tab
-             Then user will see Shipwright BuildRuns
+              And user is at namespace "aut-shipwright-build-details"
+             When user navigates to Topology in Developer perspective
+             Then user will see the workload "<workload_name>" visible
+              And user will see the Build decorator attached to "<workload_name>"
+             When user clicks on the Build decorator attached to "<workload_name>"
+             Then user will be redirected to the buildRun logs page
```

Comment:
Two When and Then can not be used in a scenario, if needed break it into two different test scenarios